tools/scripts/darwin.sh

```diff
@@ -5,7 +5,7 @@
 ARCH=x64
 INPUT_TYPE=dir
 LICENSE=MIT
-NODE_VERSION=12.16.2
+NODE_VERSION=14.15.3
```

Comment:
LTS is now version 16.  Is there any reason why we're stopping at 14?

Reply:
We are doing 14 because of the `nexe`. This is the highest/most common version of prebuilt builds of node. It just makes the creation easier/faster.